tests/main/cached_enforcer_spec.lua

```diff
@@ -32,7 +32,7 @@ describe("Cached Enforcer tests", function ()
 	    -- for ("alice", "data1", "read") will still be true, as it uses the cached result.
         e:RemovePolicy("alice", "data1", "read")
 
-        assert.is.True(e:enforce("alice", "data1", "read"))
```

Comment:
Please don't remove this.

Reply:
Fixed，I deleted it when I was debugging and forgot to put it back.